tool/build_e2e.dart

```diff
@@ -8,13 +8,18 @@ const argDevToolsBuild = 'devtools-build';
 const argUpdatePerfetto = '--update-perfetto';
 const argNoUpdateFlutter = '--no-update-flutter';
 
+// Add this flag if devtools_app does not need to be rebuilt.
+const argNoBuildApp = '--no-build-app';
+
 void main(List<String> args) async {
```

Comment:
Can you comment on `main` what items are built to make it clear what is remaining after argNoBuildApp is applied.